Factories/templates/CMakeLists.txt.tpl

```diff
@@ -9,9 +9,9 @@ include(CMSSW)
 
 # Ensure C++11 is available
```

Comment:
IIRC, there's no `-O` flags because of the impact on the build time if there's a lot of histograms to do. We can try with `-O2` first to see how it goes?